farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html

```diff
@@ -334,11 +378,13 @@ <h1 class="text-center">Seeding Report</h1>
                  * Uses deleteRecord function to delete the record in the database. Also deletes the record from the current logs that are in seedingLogs
                  */
                 deleteRowLog(logID) {
-                    let url = '/log/' + logID
-                    deleteRecord(url, this.sessionToken)
-                    for(var i=0; i < this.seedingLogs.length; i++){
-                        if(this.seedingLogs[i].id === logID){
-                            this.seedingLogs.splice(i, 1)
+                    for(let i = 0; i < logID.length; i++) {
+                        let url = '/log/' + logID[i]
+                        deleteRecord(url, this.sessionToken)
```

Comment:
deleteRecord returns a Promise.  Here the deleting of the local rows should only happen if the promise resolves successfully.  So that code should go into the then().  There should also be a catch for the error in case there is an issue with deleting the record from the DB.  Use the other API calls where we display the error banner as a guide.

Reply:
@FutzMonitor Have you look into handling possible errors during the delete here and at the other places I marked it?

Reply:
I think having the error banner appear is sufficient.  I think I may have just missed the fact that the delete was moved inside the loop and it does now have a then and a catch on it.   So this looks good to me now.